students/semko_krzysztof/lesson_04_unit_testing/test_distance.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from the_length_of_the_segment import calculate_distance
+
+
+class TestDistance(unittest.TestCase):
+    def testNoneCoordinate(self):
+        with self.assertRaises(TypeError):
+            calculate_distance(1, 5, 1, None)
+
+    def testStringCoordinate(self):
+        with self.assertRaises(TypeError):
+            calculate_distance(1, 5, 1, "aeou")
+
+    def testZeroCoordinate(self):
+        self.assertTrue(calculate_distance(0, 1, 10, 0) > 0)
```

Comment:
Tutaj sprawdzasz jedynie czy wynik jest większy od 0. Nie jest to zły test, ale skoro mamy prezycyjnie orkeślone dane wejściowe (0, 1, 10, 0), a algorytm jest deterministyczny to należy sprawdzić czy wynik, który daje jest taki prawidłowy czyli przyrównać do tego co ma wyjść.
Przy porównywaniu floatów dobrze użyć math.isclose, żeby uniknąć błędów przybliżeń https://docs.python.org/3/whatsnew/3.5.html#pep-485-a-function-for-testing-approximate-equality
Zalecam poprawkę tutaj i w pozostałych testach w tym pliku.